pkg/metadata/host.go

```diff
@@ -8,7 +8,7 @@ package metadata
 import (
 	"fmt"
 
-	"github.com/DataDog/datadog-agent/pkg/metadata/v5"
+	v5 "github.com/DataDog/datadog-agent/pkg/metadata/v5"
```

Comment:
this change does nothing I think? 🤔

Reply:
It does nothing but added automatically by my editor. Should I remove it?